modules/beachfrontBidAdapter.js

```diff
@@ -77,23 +81,56 @@ export const spec = {
         utils.logWarn(`No valid banner bids from ${spec.code} bidder`);
         return [];
       }
-      return response.map((bid) => {
-        let request = find(bidRequest, req => req.adUnitCode === bid.slot);
-        return {
-          requestId: request.bidId,
-          bidderCode: spec.code,
-          ad: bid.adm,
-          creativeId: bid.crid,
-          cpm: bid.price,
-          width: bid.w,
-          height: bid.h,
-          mediaType: BANNER,
-          currency: 'USD',
-          netRevenue: true,
-          ttl: 300
-        };
+      return response
+        .filter(bid => bid.adm)
+        .map((bid) => {
+          let request = find(bidRequest, req => req.adUnitCode === bid.slot);
+          return {
+            requestId: request.bidId,
+            bidderCode: spec.code,
+            ad: bid.adm,
+            creativeId: bid.crid,
+            cpm: bid.price,
+            width: bid.w,
+            height: bid.h,
+            mediaType: BANNER,
+            currency: 'USD',
+            netRevenue: true,
+            ttl: 300
+          };
+        });
+    }
+  },
+
+  getUserSyncs(syncOptions, serverResponses = [], gdprConsent = {}) {
+    let syncs = [];
+    let { gdprApplies, consentString } = gdprConsent;
+    let bannerResponse = find(serverResponses, (res) => utils.isArray(res.body));
+
+    if (bannerResponse) {
+      if (syncOptions.iframeEnabled) {
+        bannerResponse.body
+          .filter(bid => bid.sync)
+          .forEach(bid => {
+            syncs.push({
+              type: 'iframe',
+              url: bid.sync
```

Comment:
not adding gdpr params?

Reply:
The gdpr params will be added by the server in this case.